barretenberg/cpp/src/barretenberg/commitment_schemes/ipa/ipa.hpp

```diff
@@ -533,7 +562,8 @@ template <typename Curve_> class IPA {
 
         ASSERT(ipa_relation.get_value() == -opening_claim.commitment.get_value());
         // This should return an actual VerifierAccumulator
-        return {round_challenges_inv, G_zero};
+        Fr stdlib_log_poly_length(static_cast<uint256_t>(log_poly_length));
```

Comment:
insecure. Maybe we can add a constraint that 1<<log_poly_length is poly_length, but its also currently broken in sumcheck...

Reply:
Worth adding a comment / linking a TODO if one exists? or making one if it doesn't?